html/changelogs/geeves-cult_additions.yml

```
@@ -0,0 +1,9 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - bugfix: "Fixed Daemon Forges not properly cultifying items."
+  - tweak: "Reduced the damage cult swords do by five."
```

This means nothing to players. 

```suggestion
  - tweak: "Reduced the damage cult swords do by 1/6th."
```